Automattic/jetpack
# Boost: Do not show performance improvement if no modules are active.

#### Changes proposed in this Pull Request:
* Converted `Jetpack_Boost` class to a singleton.
* Return `null` for noBoost scores if no modules are active.

#### Jetpack product discussion
p1633516006013600/1633441358.008100-slack-C0232B8CUHG

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Take a score with boost modules active.
* Check in the network tab on the browser's developer tool to see the response, it should include a `noBoost` score. for the `/score-speed` requests.
*Disable all the modules and take speed score again.
* `noBoost` from the request should now be `null`.
